docs/dev-guide/phplang/features.mdx

```diff
@@ -485,6 +485,7 @@ $count = $workflow->addGreeting("World");
 
 In Workflow Update methods, all Workflow features are available, such as executing Activities and child Workflows, and waiting on timers/conditions.
 In cases where it's known that the update will take a long time to execute, or you are not interested in the outcome of its execution, you can use the stub method [`startUpdate`](https://php.temporal.io/classes/Temporal-Client-WorkflowStubInterface.html#method_startUpdate) and move on immediately after receiving the validation result.
+Note that the processing Workflow worker must be available. Otherwise, the request may block indefinitely or fail due to a timeout.
```

Comment:
```suggestion
Note that the processing Workflow Worker must be available.
Otherwise, the request may block indefinitely or fail due to a timeout.
```